sdktests/server_side_events_users.go

```diff
@@ -72,17 +71,18 @@ func doServerSideEventUserTests(t *ldtest.T) {
 
 	flagValue := ldvalue.String("value")
 	defaultValue := ldvalue.String("default")
-	users := NewUserFactory("doServerSideEventUserTests",
-		func(ub lduser.UserBuilder) {
-			ub.FirstName("first").LastName("last").Country("us").
-				Custom("preferredLanguage", ldvalue.String("go")).Custom("primaryLanguage", ldvalue.String("go"))
+	contexts := data.NewContextFactory(
+		"doServerSideEventUserTests",
+		func(b *ldcontext.Builder) {
+			b.SetString("firstName", "first").SetString("lastName", "last").SetString("country", "us").
+				SetString("preferredLanguage", "go").SetString("primaryLanguage", "go")
```

Comment:
This data is a bit obsolete because those attribute names have no special meaning any more, but I'll address that in a bigger update to these tests.